packages/eds-core-react/src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -128,7 +128,7 @@ export const LongListWithTooltips: Story<TooltipProps> = () => {
       {items.map((_, i) => (
         // eslint-disable-next-line react/no-array-index-key
         <span key={i}>
-          <Tooltip title={`Icon ${i}`} placement="right">
+          <Tooltip title={`Icon ${i}`} placement="right" aria-live="polite">
```

Comment:
You can remove these `aria-live`, they do nothing because they have to exist outside of and before the elements added (tooltip is conditionally rendered)